src/js/components/award/shared/activity/ContractGrantsActivityChart.jsx

```diff
@@ -290,29 +298,62 @@ const ContractGrantsActivityChart = ({
                     ticks={xTicks}
                     scale={xScale}
                     line />
-                {/* These will be removed in the next PR  */}
-                {/* {xScale && <VerticalLine
+                {/* start line */}
+                {xScale && <VerticalLine
                     xScale={xScale}
                     y1={-10}
                     y2={height}
                     textY={0}
-                    text="Start Graph"
+                    text="Start"
                     xMax={xDomain[1]}
                     xMin={xDomain[0]}
                     xValue={xDomain[0]}
-                    showTextPosition="top"
-                    adjustmentX={padding.left} />}
+                    showTextPosition="right"
+                    adjustmentX={padding.left}
+                    textClassname="vertical-line__text start"
+                    lineClassname="vertical-line start" />}
+                {/* today line */}
                 {xScale && <VerticalLine
                     xScale={xScale}
                     y1={-10}
                     y2={height}
                     textY={0}
-                    text="End Graph"
+                    text="Today"
+                    xMax={xDomain[1]}
+                    xMin={xDomain[0]}
+                    xValue={Date.now()}
+                    showTextPosition="left"
+                    adjustmentX={padding.left}
+                    textClassname="vertical-line__text today"
+                    lineClassname="vertical-line today" />}
+                {/* end line */}
+                {xScale && dateEndLine && <VerticalLine
+                    xScale={xScale}
+                    y1={-10}
+                    y2={height}
+                    textY={0}
+                    text={endLineText}
+                    xMax={xDomain[1]}
+                    xMin={xDomain[0]}
+                    xValue={dateEndLine}
+                    showTextPosition="left"
+                    adjustmentX={padding.left}
+                    textClassname={`vertical-line__text ${endLineClassName}`}
+                    lineClassname={`vertical-line ${endLineClassName}`} />}
+                {/* potential end line */}
+                {xScale && datePotentialEndLine && <VerticalLine
```

Comment:
Recommenmd introducing case where `datePotentialEndLine` and `dateEndLine` are the same!

Reply:
Yeah, that is an edge case we have in Dev-4553